src/cmdstan-guide/laplace_sample_config.Rmd

```diff
@@ -0,0 +1,105 @@
+# Laplace sampling
+
+The `laplace` method produces a sample from a normal approximation
+centered at the mode of a distribution.
+If the mode is a maximum a posteriori (MAP) estimate,
+the samples provides an estimate of the standard deviation
+of the posterior distribution.
+If the mode is a maximum likelihood estimate (MLE),
+the sample provides an estimate of the standard error of the likelihood.
+This is computationally inexpensive compared to exact Bayesian inference with MCMC.
+The goodness of this estimate depends on both the estimate of the mode
+and how much the true posterior resembles a Gaussian.
```

Comment:
I would change this to
"The goodness of this estimate depends on both the estimate of the mode
and how much the true posterior in the unconstrained space resembles a Gaussian."